appveyor.yml

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
Since the `README.md` claims that the module is not compatible with Windows, we can probably add `appveyor.yml` to the list of files to delete in `.sync.yml`.